resourcet/Control/Monad/Trans/Resource.hs

```diff
@@ -106,6 +107,20 @@ allocate :: MonadResource m
          -> m (ReleaseKey, a)
 allocate a = liftResourceT . allocateRIO a
 
+-- | Perform some allocation where the return value is not required, and
+-- automatically register a cleanup action.
+--
+-- @allocate_@ is to @allocate@ as @bracket_@ is to @bracket@
+--
+-- This is almost identical to calling the allocation and then
+-- @register@ing the release action, but this properly handles masking of
+-- asynchronous exceptions.
```

Comment:
Please add a `@since` comment, a changelog entry, and a minor version bump in the cabal file

Reply:
Done, thanks!